packages/spec/src/spec.emu.html

```diff
@@ -283,12 +283,15 @@ <h1>Syntactic Grammar</h1>
     `using` IdentifierOrMemberExpression `;`
 
 ModelStatement :
+    DecoratorList? `model` Identifier TemplateParameters? IsModelHeritage `;`
     DecoratorList? `model` Identifier TemplateParameters? ModelHeritage? `{` ModelBody? `}`
```

Comment:
Don't you still need to say that ModelHeritage is either IsModelHeritage or ExtendsModelHeritage somewhere?

Reply:
I thought the spec generation used to fail on a missing reference, but it's green now.

Reply:
Really? What about `model X is Y { ... }`. That's still valid, right?

Reply:
Hahah you are right this change is nonsense.